dist/com.nvidia.spark.rapids.SparkShimServiceProvider.sparkNonSnapshot

```diff
@@ -0,0 +1,6 @@
+com.nvidia.spark.rapids.shims.spark301.SparkShimServiceProvider
```

Comment:
Note hardcoded these for now, hopefully can make this smarter and generate on the fly later.